mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorRewriting.cpp

```diff
@@ -453,8 +453,15 @@ struct ReshapeRewriter : public OpRewritePattern<ReshapeOp> {
       auto rtp = getRankedTensorType(op.getResult());
       auto denseTp =
           RankedTensorType::get(rtp.getShape(), rtp.getElementType());
-      auto reshape = rewriter.create<ReshapeOp>(loc, denseTp, op.getSrc(),
-                                                op.getReassociation());
+      ReshapeOp reshape;
+      if constexpr (std::is_same<ReshapeOp, tensor::ExpandShapeOp>::value) {
```

Comment:
Same here. Use `{` `}` for multi-line statements.

Reply:
Did you mean some other if stmt, as this one already have.